### PR TITLE
new icon: nano (original, original-wordmark)

### LINKED ISSUE #1279: [ICON REQUEST]: nano
### I have searched through the issues and didn't find my problem.

- [X] Confirm

### About the icon

GNU text editor in the terminal

### Links and sources

[Website](https://www.nano-editor.org/)

### Images

[PNG](https://commons.wikimedia.org/wiki/File:GNU-Nano-Logo.png)
image

[PNG Wordmark (not official)](https://www.maketecheasier.com/mastering-nano-text-editor/)
image

[SVG](https://commons.wikimedia.org/wiki/File:Gnu-nano.svg)
SVG doesn't show properly for some reason, so click the link to see it
image

### Additional information

The original logo is just made out of ASCII art, so could be hard to find a good SVG for it.

### Original logo from the website

```
                :::                           The                 
  iLE88Dj.  :jD88888Dj:                                           
.LGitE888D.f8GjjjL8888E;        .d8888b.  888b    888 888     888 
iE   :8888Et.     .G8888.      d88P  Y88b 8888b   888 888     888 
;i    E888,        ,8888,      888    888 88888b  888 888     888 
      D888,        :8888:      888        888Y88b 888 888     888 
      D888,        :8888:      888  88888 888 Y88b888 888     888 
      D888,        :8888:      888    888 888  Y88888 888     888 
      D888,        :8888:      Y88b  d88P 888   Y8888 Y88b. .d88P 
      888W,        :8888:       "Y8888P88 888    Y888  "Y88888P"  
      W88W,        :8888:                                         
      W88W:        :8888:      88888b.   8888b.  88888b.   .d88b. 
      DGGD:        :8888:      888 "88b     "88b 888 "88b d88""88b
                   :8888:      888  888 .d888888 888  888 888  888
                   :W888:      888  888 888  888 888  888 Y88..88P
                   :8888:      888  888 "Y888888 888  888  "Y88P" 
                    E888i                                         
                    tW88D             Text Editor Homepage        
```

### PR DESCRIPTION
**Pull request by Snailedlt**
## Double check these details before you open a PR

- [x] PR does not match another non-stale PR currently opened
- [x] PR name matches the format *new icon: <i>Icon name</i> (<i>versions separated by comma</i>)*. More details [here](https://github.com/devicons/devicon/blob/develop/CONTRIBUTING.md#overview)
- [x] PR's base is the `develop` branch.
- [x] Your icons are inside a folder as seen [here](https://github.com/devicons/devicon/blob/develop/CONTRIBUTING.md#organizational-guidelines)
- [x] SVG matches the standards laid out [here](https://github.com/devicons/devicon/blob/develop/CONTRIBUTING.md#svgStandards)
- [x] A new object is added in the `devicon.json` file as seen [here](https://github.com/devicons/devicon/blob/develop/CONTRIBUTING.md#-updating-the-deviconjson-)

**This PR closes #1279**

## Link to prove your SVG is correct and up-to-date.

https://www.nano-editor.org/

### CODE REVIEW

File: devicon.json [hunk 1]
```diff
@@ -4073,6 +4073,40 @@
...
         "aliases": []
     },
+    {
+        "name": "nano",
+        "altnames": [
+            "gnu-nano",
+            "GNU nano"
+        ],
+        "tags": [
+            "text editor",
```
Comment (Panquesito7) [inline devicon.json:4083]: I'm not sure tags should have spaces. 🤔


__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :question: I have a question

File: icons/nano/nano-original.svg [hunk 2]
```diff
@@ -0,0 +1 @@
....78-.22.201-.505.302-.855.302s-.635-.102-.854-.305a1.026 1.026 0 0 1-.328-.789c0-.197.052-.373.157-.526a.994.994 0 0 1 .444-.352zm-.124.878a.67.67 0 0 0 .193.499c.129.128.3.192.512.192.215 0 .387-.062.515-.186a.652.652 0 0 0 .193-.494.646.646 0 0 0-.197-.489.712.712 0 0 0-.511-.187.715.715 0 0 0-.508.184.629.629 0 0 0-.197.481zm.029-1.729c0 .192.063.349.19.471a.673.673 0 0 0 .486.183.67.67 0 0 0 .489-.185.634.634 0 0 0 .19-.476.61.61 0 0 0-.19-.461.68.68 0 0 0-.489-.182.672.672 0 0 0-.488.182.626.626 0 0 0-.188.468zM106.011 125.904H105.421c-.072 0-.125-.017-.161-.052-.036-.035-.054-.087-.054-.157 0-.075.021-.128.062-.157s.12-.044.237-.044h.052v-2.616h-.046c-.116 0-.196-.015-.24-.046-.044-.031-.065-.085-.065-.163 0-.072.018-.124.054-.158.036-.034.091-.051.167-.051h1.226c.482 0 .866.155 1.153.466.286.311.43.73.43 1.259 0 .257-.041.496-.123.717s-.202.414-.359.581a1.282 1.282 0 0 1-.53.33c-.208.069-.476.103-.803.103-.039 0-.104-.002-.196-.006a5.345 5.345 0 0 0-.214-.006zm0-.409a3.882 3.882 0 0 0 .363.017c.476 0 .825-.106 1.047-.319.222-.213.332-.549.332-1.008 0-.461-.111-.796-.334-1.006-.223-.21-.579-.315-1.068-.315-.091 0-.16.001-.208.003a1.293 1.293 0 0 0-.132.012v2.616z"/></g></svg>
```
Comment (Panquesito7) [inline icons/nano/nano-original.svg:1]: Missing an endline. Without this endline, there could be issues with some IDEs and configurations.
```suggestion
}

```


__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix&nbsp; | &nbsp;**Tags:** Inefficient

Comment (Snailedlt) [reply icons/nano/nano-original.svg:1]: We haven't been enforcing this earlier, should we add it to the SVG Standards too?

File: devicon.json (hunk 1 above)
Comment (Snailedlt) [reply devicon.json:4083]: We have a lot of tags with spaces already. Is there any reason we shouldn't have spaces?

**Comment by Snailedlt** [comment]
The devicon.json check is failing. Any ideas why?

**Comment by Panquesito7** [comment]
> The devicon.json check is failing. Any ideas why?

I think I made a mistake in my suggestion. 👀
I'll fix it right away.

**Comment by Snailedlt** [comment]
Ahh, okey. Looks good now 😊👌

**Review by Panquesito7 — APPROVED** [review]
Looking great. Thanks! 🚀


__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :trophy: This code is awesome&nbsp; | &nbsp;**Tags:** Maintainable

**Pull request merged**